File: ClientApp/src/app/models/usuarioOrganizacion.ts
import { IBase } from "./base";
import { IOrganizacion } from "./organizacion";
import { IUsuario } from "./usuario";

export interface IUsuarioOrganizacion extends IBase{
    usuarioId: number
    usuario: IUsuario
    organizacionId:number
    organizacion:IOrganizacion
}

File: Models/UsuarioOrganizacion.cs
using System.Text.Json.Serialization;

namespace Donaciones.Models
{
    public class UsuarioOrganizacion:Base
    {
        public int UsuarioId { get; set; }
        [JsonIgnore]
        public Usuario Usuario { get; set; }
        public int OrganizacionId { get; set; }
        public Organizacion Organizacion { get; set; }
    }
}
